test/edu/csus/ecs/pc2/core/log/NullController.java

```diff
@@ -654,6 +654,13 @@ public void submitRun(ClientId submitter, Problem problem, Language language, Se
         
     }
 
+    @Override
+    public void submitRun(ClientId submitter, Problem problem, Language language, String entry_point, SerializedFile mainFile, SerializedFile[] auxFiles, long overrideTimeMS,
+            long overrideSubmissionID) {
+        // TODO Auto-generated method stub
```

Comment:
If there is nothing to do/follow up on, remove TODO statement

Reply:
Fixed.

Reply:
I'd just like to point out that there are currently **_107_** methods in just this class that contain the `//TOD Auto-generated method stub` comment.  Seems a bit picky to ask for _one_ of them to be removed...

Reply:
I disagee that it is picky to remove them, it is an opportunity to remove unsed TODO items.   The alternative as cleanup is to review every single //TODO Auto-generated, if we can reduce that or provide an actual TODO then there will be less work for the team to do later.
Further removing auto generated TODO statements should be considered a habit, in order to remove them when one knows their context.   Again less work later to delete/update one line.